sbin/prepareWorkspace.sh

```diff
@@ -591,10 +591,14 @@ checkingAndDownloadingFreeType() {
     if ./configure --help | grep "with-png"; then
       pngArg="--with-png=no"
     fi
+    local harfbuzzArg=""
+    if ./configure --help | grep "with-harfbuzz"; then
```

Comment:
I would definitley call it with interpreter - `sh ./configure` to be sure it passes also from various custom tarballs.

Reply:
If we do this we should do it on line 591 as well. So this seems prior art. Since the final `configure` call of the `freetype` sources is using `bash ./configure` I'd suggest to use that consistently when calling `--help` as well.